### ISSUE TITLE
unknown shorthand flag: 'd' in -d pixel_6

### ISSUE THREAD

**Issue by jfaris** [CircleCI-Public/android-orb#104]
### Orb version

3.0.0

### What happened

After updating to version 3 of this orb, we are seeing this error:

```
Error: unknown shorthand flag: 'd' in -d pixel_6
ERRO[0000] Exit with status 1 due to generic error       error="unknown shorthand flag: 'd' in -d pixel_6"
```

This is our complete config for the `start_emulator_and_run_tests` step:

```yaml
      - android/start_emulator_and_run_tests:
          avd_name: Pixel_6_API_31
          additional_avd_args: -d pixel_6
          system_image: system-images;android-31;google_apis;x86_64
          max_tries: 1
          run_tests_working_directory: .
          post_emulator_launch_assemble_command: |
            export SENTRY_DISABLE_AUTO_UPLOAD=true
            npx detox build -c android.sim.staging
          test_command: |
            npx detox test -c android.sim.staging --loglevel verbose --record-videos failing --artifacts-location /tmp/failed-detox-tests
```

### Expected behavior

Tests should run as expected. It worked on version 2.5.0 and earlier, and started failing with v3.

**Comment by marboledacci** [comment]
This should be solved on version 3.0.1

**Comment by jfaris** [comment]
@marboledacci Thank you for the quick response on this. I can confirm that 3.0.1 fixes the issue for us.
